examples/source-rouge.adoc

```diff
@@ -39,3 +40,20 @@ fn main() {
     println!("Hello stretched World!");
 }
 ----
+
+== Rouge Reatures
+
+// TODO docinfo not loaded
```

Comment:
This sounds more concerning...

Reply:
docinfo is there now thanks to the rebase on top of #324. However, the class `hll` is missing from the styles inlined by Asciidoctor. Again, anything else than highlight.js is not getting my attention right now.